include/open62541pp/types/Variant.h

```diff
@@ -507,11 +516,21 @@ struct VariantHandler<VariantPolicy::Copy> {
         var.setScalarCopy(value);
     }
 
+    template <typename T, typename = std::enable_if_t<!std::is_reference_v<T>>>
```

Comment:
yeah, it is really hard to see why this `std::enable_if` is needed, just by looking at the code.

Reply:
Totally agree.. let leave it as a draft PR.